python/ray/_private/runtime_env/container.py

```diff
@@ -33,10 +34,13 @@ async def setup(
             "--network=host",
             "--pid=host",
```

Comment:
Q: is shm shared across all containers?

Reply:
I believe it shares memory through the option `--ipc=host` (although I'm not super familiar with the details)

Reply:
hmm we may need tests.

Reply:
With `--ipc=host` containers will have access to host's shared memory (ie created outside of any container on the host itself)